flutter/lib/sentry_flutter.dart

```diff
@@ -1,3 +1,4 @@
+// ignore: invalid_export_of_internal_element
```

Comment:
Why is this needed if `sentry_screenshot_widget` is public?

Reply:
<img width="679" alt="Bildschirmfoto 2022-11-07 um 13 54 21" src="https://user-images.githubusercontent.com/3984453/200315524-f8a0879a-8c7f-4169-a53a-be0c2cb58210.png">